config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Is this intentional? 

We were always reverting the change in this. How is the image version set for release? Does it work with `make reset` ?